docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,726 @@
+'Ultimate' edition. Since you're a student, you aren't limited to the 30-day trial that would otherwise
+be applied.
+
+Once you've downloaded and installed it (you'll probably need to create an account with your student 
+email), open it so that the welcome page shows up.
+
+>Note: some people prefer to use text editors like [Microsoft's VSCode](https://code.visualstudio.com/) or
+>[GitHub's Atom](https://atom.io/) instead of IntelliJ for working on the frontend. We'll be focusing
+>on using IntelliJ for all of this, but feel free to try those out if you're interested.
+
+### Cloning the repositories with Git
+
+Now that you've gotten IntelliJ set up, we'll get to cloning the repositories on your computer.
+
+>Note: you may need a GitHub account to do this. If you encounter an error trying to clone it,
+>then then just sign up and and retry the steps after logging into GitHub in IntelliJ.
+
+Inside of IntelliJ, go to File > New... > Project From Version Control..., make sure the 
+`Version Control` box is set to `Git`, paste in the following links, and choose where you 
+want them to be downloaded to. 
+
+>Note: if you're on Windows, you may also need to install git (it should give you a button to do so)
+>to be able to clone the projects.
+
+The links you'll want to use are:
+- Frontend: `https://github.com/Code-4-Community/jumpstart-frontend.git`
+- Backend: `https://github.com/Code-4-Community/jumpstart-backend.git`
+
+After the projects are open, you'll want to view the starter code by entering the following command into the terminal
+at the bottom of your IntelliJ window for both the frontend and backend projects.
+
+```
+git checkout starter-code
+```
+
+If you don't run the above command, then what you'll probably be working with is a later-stage version of the project.
+That's alright, and you're free to use that later-stage version for help, inspiration, or even if you don't want to
+do the project but are still interested in seeing examples of it. 
+
+Once you've done that, you're ready to install other programs we'll need and build the projects. If you've encountered
+any issues, please don't hesitate to reach out to us so we can help. It's really common to run into errors
+when setting up a new environment.
+
+### Installing required apps
+
+There are a few other programs you'll need to install before we're done here. 
+
+For the frontend and backend, open up the projects and start reading the README.md files by 
+double clicking them in the project pane on the upper left. If you don't see it, you may have to
+expand the pane by clicking on the vertically spelled "project" tab, and then opening up
+the dropdowns indicating folders.
+
+To make sure the formatted text pops up, once you open the README.md files, click on the image-like 
+icon in the upper right to view the formatted markdown. You can also try viewing it side-by-side as
+plaintext and formatted markdown if you're interested to see what it looks like.
+
```

Comment:
You have to add a sentence to tell them to follow the instructions in the READMEs

Reply:
It sort of says that above, but I'll reiterate.